src/ObjectGraph.es6.js

```diff
@@ -62,15 +62,17 @@ ObjectGraph.prototype.types = ObjectGraph.types = {
 };
 
 // Never visit/store these object keys.
-ObjectGraph.prototype.blacklistedKeys = [ uid.key, uid.key__, '__proto__' ];
+ObjectGraph.prototype.blacklistedKeys = [ uid.key, uid.key__, '__proto__',
+  '$UID', '$UID__']; // Foam assigns unique Ids to objects.
```

Comment:
Please clarify that this refers to the last two keys in the list. Also, please add a TODO to upstream id-js to FOAM so we don't have two unique identifiers floating around.